package.json

```diff
@@ -47,6 +47,7 @@
     "@formatjs/intl-numberformat": "^6.2.5",
     "@formatjs/intl-pluralrules": "^4.0.13",
     "@gorhom/portal": "^1.0.14",
+    "@microlink/mql": "^0.10.32",
```

Comment:
Oh this is cool - I imagine/think we want to use some proxy.

Reply:
Removed in favour of API.read() and proxy